core/controller/src/main/resources/application.conf

```diff
@@ -38,3 +38,13 @@ spray.can.server {
     max-content-length = 50m
   }
 }
+
+# Disable legacy metrics in akka-cluster.
```

Comment:
Thats not the final version of the config, going to document it we indeed turn it off.